scripts/setup/rewrite_yaml_files.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+
+pushd $HOME/vhive/configs/knative_yamls >/dev/null
+
+rm -rf serving-core-firecracker.yaml
+rm -rf serving-crds.yaml
+rm -rf serving-default-domain.yaml
+rm -rf serving-hpa.yaml
+rm -rf serving-post-install-jobs.yaml
+rm -rf serving-storage-version-migration.yaml
+rm -rf ../calico/canal.yaml
+rm -rf ../metallb/metallb-ipaddresspool.yaml
+rm -rf ../setup/kube.json
+
+wget -q https://github.com/knative-extensions/net-istio/releases/download/knative-v1.9.0/net-istio.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-core.yaml && mv serving-core.yaml serving-core-firecracker.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-core.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-crds.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-default-domain.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-hpa.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-post-install-jobs.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-storage-version-migration.yaml
+wget -q https://raw.githubusercontent.com/vhive-serverless/vHive/main/configs/calico/canal.yaml
```

Comment:
I think we should just use the one that is already on the node within the vhive repo. No need to download the same file from github.